examples/archive.php

```diff
@@ -27,13 +27,14 @@
     });
 });
 
-$tar->on('error', function ($e = null) {
+$tar->on('error', function (Exception $e) {
     // should not be invoked, unless the stream is somehow interrupted
-    echo 'ERROR processing tar stream' . PHP_EOL . $e;
+    echo 'ERROR processing tar stream' . $e->getMessage() . PHP_EOL;
 });
-$stream->on('error', function ($e = null) {
+
+$stream->on('error', function (Exception $e) {
```

Comment:
In other examples you changed the same error message to `echo 'Error: ' . $e->getMessage() . PHP_EOL;`.
Any reason why this one is different?